src/dpos/components/RegisterCandidateForm.vue

```diff
@@ -78,8 +78,9 @@
                 </b-input-group>
               </b-col>
             </b-row>
+            <p class="mb-3 float-right" style="font-size: 12px; color:red" v-if="!validFee">({{ $t("components.registerCandidate.validate_fee", {min: state.dpos.minCandidateFee}) }})</p>
```

Comment:
`minCandidateFee` is in basis points, but currently 1000 is displayed as 1000%, not 10%.